src/components/tools/drawing-tool/color-swatch.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from "classnames";
+import React from "react";
+import SwatchCheckIcon from "../../../clue/assets/icons/drawing/check-icon.svg";
+import NoColorIcon from "../../../clue/assets/icons/drawing/no-color-icon.svg";
+import { isLightColorRequiringContrastOffset } from "../../../utilities/color-utils";
+
+export enum SwatchColor {
+  none = "none",
+  black = "#000000",
+  white = "#ffffff",
+  red = "#eb0000",
+  green = "#008a00",
+  blue = "#0000ff",
+  gray = "#bfbfbf",
+  orange = "#ff8415",
+  yellow = "#ffff00",
+  purple = "#d100d1"
```

Comment:
I think most of these are defined in `vars.scss`.  Not sure it's worth the effort to import them from there.

Reply:
I thought about it but never quite got up the motivation to do it. It probably would make sense, however, both for the colors and for some of the sizing constants.